src/style.css

```diff
@@ -0,0 +1,101 @@
+@font-face {
+  font-family: "Avenir-heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: "Avenir-book";
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-weight: 300;
+}
+
+.big-form,
+.small-form {
+  margin-top: 20px;
+  position: relative;
+}
+
+.input {
+  width: 100%;
+  height: 70px;
+
+  background: #fff;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  border-radius: 4px;
+
+  font-family: "Avenir-book", sans-serif;

```

Comment:
add `font-family: "Avenir-book"` to html and here use `inherit`
https://developer.mozilla.org/en-US/docs/Web/CSS/inherit